### PR TITLE
add a user with node add_user

### PR DESCRIPTION
**Pull request by Cinezaster**
Because this will be deployed on a server a script to create a user.json file would be nice.

### CODE REVIEW

**Comment by GAMEBOOK** [comment]
Good migration commit :) GG

users.json = Top! Purpose When [npm test]>

[Fri May January 2015 0:22:37] INFO: Loaded users
[Fri May January 2015 0:22:37] INFO: Started: Forever Web Console
[Fri May January 2015 0:22:37] INFO: Server listening on port: 8085

http://gyazo.com/ad04e4be519aac739a054dc41bd2e9f9

File: add_user.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+rl.on('line', function(line){
+	switch (ticker) {
+		case 0:
+			user.username = line;
+			console.log(colors.yellow('Enter the password:'));
+			
+			break;
+		case 1:
+			user.password = line;
```
Comment (FGRibreau) [inline add_user.js]: password should not be stored in clear form (maybe use sha256 at least ?)

**Comment by Cinezaster** [comment]
password SHA256 encryption + unique uuid's for the id's

**Comment by FGRibreau** [comment]
@Cinezaster released in v0.3.0

**Pull request merged**